convert2rhel/unit_tests/subscription_test.py

```diff
@@ -146,6 +146,42 @@ def __call__(self, *args, **kwargs):
         return [namedtuple("Sub", ["pool_id", "sub_raw"])("samplepool", "Subscription description")]
 
 
+class PromptUserMocked(unit_tests.MockFunction):
+    def __call__(self, *args, **kwargs):
+        return True
+
+
+class TestCheckNeededReposAvailability(object):
+    def test_check_needed_repos_availability(self, monkeypatch, caplog):
+        monkeypatch.setattr(utils, "ask_to_continue", PromptUserMocked())
+        monkeypatch.setattr(subscription, "get_avail_repos", lambda: ["rhel_x", "rhel_y"])
+
+        avali_repos_message = "Needed RHEL repositories are available."
+        subscription.check_needed_repos_availability(["rhel_x"])
+
+        assert avali_repos_message in caplog.records[-1].message
+
+        no_avali_repos_message = (
+            "Some repositories are not available: rhel_z. Some packages may not be replaced with"
+            "their corresponding RHEL packages when converting."
+        )
```

Comment:
Isn't this test message now out of date? (Also, "withtheir".)

Reply:
Yes now looking at it I do belive that this message is outdated I will update the message and fix the "withtheir" issue.